.github/workflows/staging-apps.yaml

```diff
@@ -0,0 +1,58 @@
+name: staging-apps
+
+on:
+  push:
+    branches:
+      - 'staging/ideaspace'
+      - 'staging/ideaspace[1-2]'
+      - 'staging/recruitment'
+      - 'staging/recruitment[1-2]'
+      - 'staging/projects'
+      - 'staging/projects[1-2]'
+      - 'staging/website'
+      - 'staging/website[1-2]'
```

Comment:
Pushing to `staging/ideaspace` will trigger a deployment where the link you'll see the results in is https://ideaspace.pages.dev/ , This is your team's staging environment!

For A/B testing you can merge the A testing to `staging/ideaspace1` and B to `staging/ideaspace2` and the url you'll see the results in is https://ideaspace1.pages.dev/ or https://ideaspace2.pages.dev/ depending on the branch you pushed to